pkg/kv/kvserver/store_raft.go

```diff
@@ -738,6 +741,7 @@ func (s *Store) raftTickLoop(ctx context.Context) {
 			if s.cfg.NodeLiveness != nil {
 				s.updateLivenessMap()
 			}
+			s.updateStoreOverloadMap()
```

Comment:
Have we measured the latency of this? Although I suppose it would get absorbed by the ticker.

Reply:
This should be really cheap, especially compared to the cost of ticking multiple replicas. Anything particular in `updateStoreOverloadMap` that worries you? I suppose if you had many more stores than unquiesced replicas it might end up costing about the same but honestly I doubt it.

Reply:
Nah, I think this is fine since the ticker will absorb any latency. But otherwise, the tick path seems hot enough to warrant a modicum of scrutiny, although the store tick less so.